pkgs/tools/misc/beam-book/add-HEAD-to-git-shortlog.patch

```diff
@@ -0,0 +1,7 @@
+--- a/bin/gitlog.sh	1969-12-31 18:00:01.000000000 -0600
++++ b/bin/gitlog.sh.fix	2021-07-03 12:07:09.851636280 -0500
+@@ -1,3 +1,3 @@
+ #!/nix/store/ca5hi4m78vhqarchh9p71pl98lbpbh2f-bash-interactive-4.4-p23/bin/bash
```

Comment:
That will most likely change in the next staging run.

Reply:
Not quite sure what you mean by this change. Mind explaining?

Reply:
The patch will likely not apply if the path to /bin/bash changes which happens often in staging.

Reply:
Ahh. That was not quite clear from the review block. A bit of a moot point at this point though.

I upstreamed the change this patch implements to the source repository, so the script should just work now.
I can change this particular commit to either the latest release, or a pre-release version of the book, whichever is better.

Reply:
I think either is probably fine.